### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by cod3burrito**

### CODE REVIEW

File: views/indivpost.handlebars [hunk 1]
```diff
@@ -1,8 +1,13 @@
 <article class="card">
     <div class="card-header flex-row justify-space-between">
-      <h2>{{userPost.typeOfDrone}}</h2>
-       <span>{{userPost.tags}}</span>
+      <span><img src="{{image}}" class="card-img-top" alt="...">
```
Comment (Ebyington) [inline views/indivpost.handlebars:3]: {{userPost.image}}

**Review by Ebyington — CHANGES_REQUESTED** [review]
image link

**Comment by cod3burrito** [comment]
changes made

**Review by Ebyington — CHANGES_REQUESTED** [review]
cap L

**Review by Ebyington — APPROVED** [review]

**Pull request merged**